### DISCUSSION TITLE
Install BepInEx in Synology for ValheimPlus

### DISCUSSION THREAD

**Discussion by MAL0W** [lloesche/valheim-server-docker#291] [Q&A]
Could someone please help me install BepInEx on my synology docker?

**Comment by InB4DevOps** [comment]
Could you please elaborate which problem you're facing exactly? 
You don't need to install BepInEx separately.

**Reply by MAL0W** [reply]
Well, I need to install BepInEx to be able to run ValheimPlus.

**Reply by lloesche** [reply]
https://github.com/lloesche/valheim-server-docker#valheimplus

**Reply by MAL0W** [reply]
Thanks and to install this mod I need to install BepInEx but I can't find by google or youtube how to do it for synology

https://www.nexusmods.com/valheim/mods/574

**Reply by MAL0W** [reply]
Ok, I saw that yesterday you added compatibility with BepInEx in your container, I had to delete the docker and create it again so that it would create the /config/bepinex/ folder and there inside the plugins folder I put my plugin dlls but it still does not activate me No mods that I add when I start the game, I'll be watching the other users to see what they say.

One thing, I read you that the plugins in the /config/bepinex/ folder are copied to the /opt/valheim/bepinex/BepInEx/plugins/ folder but in mods like this one https://www.nexusmods.com/valheim/mods/90 that has a folder in addition to a dll file, this folder is not autocopy.

Sorry if I have grammar mistakes, my English is not very good.
Thank you very much again.

**Reply by lloesche** [reply]
If all you want is ValheimPlus you don't need BepInEx. All you do is set the variable `VALHEIM_PLUS=true` as described in the README section I linked above and you're done.

> but in mods like this one https://www.nexusmods.com/valheim/mods/90 that has a folder in addition to a dll file, this folder is not autocopy

Good catch! I just pushed an update that makes sure to copy everything in the `plugins/` folder.

**Reply by Schlicki** [reply]
Can you switch to the furthermore supported fork? https://thunderstore.io/c/valheim/p/Grantapher/ValheimPlus_Grantapher_Temporary/
